collect_app/build.gradle

```diff
@@ -389,6 +381,7 @@ dependencies {
     testImplementation Dependencies.androidx_test_core_ktx
 
     androidTestImplementation project(':androidtest')
+    androidTestImplementation project(':test-forms')
```

Comment:
Should we move other resources like media files there too?

Reply:
Yeah sounds good! I'll keep the instances and dbs in where they are for now, but move all the media files so everything for the test forms is in one place.